execution/executor/src/chunk_executor/mod.rs

```diff
@@ -595,10 +595,14 @@ impl<V: VMBlockExecutor> ChunkExecutorInner<V> {
             .map(|t| t.into())
             .collect::<Vec<SignatureVerifiedTransaction>>();
 
+        let mut extra_info = Vec::new();
+        // TODO(grao): Pass in persisted extra info.
+        extra_info.resize(txns.len(), ExtraInfo::new_empty());
         // State sync executor shouldn't have block gas limit.
         let execution_output = DoGetExecutionOutput::by_transaction_execution::<V>(
             &V::new(),
             txns.into(),
+            extra_info,
```

Comment:
It might be better to keep SignatureVerifiedTransaction and ExtraInfo together, instead of two separate vectors?